api/audit/permissions.py

```diff
@@ -4,6 +4,7 @@
 
 from organisations.models import Organisation
 from organisations.permissions.permissions import VIEW_AUDIT_LOG
```

Comment:
This should be imported from `projects.permissions`

Reply:
Done